src/features/events/l10n/messageIds.ts

```diff
@@ -1,6 +1,14 @@
 import { m, makeMessages } from 'core/i18n';
 
 export default makeMessages('feat.events', {
+  addPerson: {
+    addButton: m('Add person'),
+    addPlaceholder: m('Start typing to add participant'),
+    status: {
+      booked: m('Already booked'),
+      signedUp: m('Already Signed up'),
```

Comment:
I think "Already signed up" should just be "Signed up". When it says "Already signed up", I think it signals that it's not possible to select this person, but it is. All other strings are fine IMO.